src/_locales/de.json

```diff
@@ -0,0 +1,94 @@
+{
+  "audio-device-communications": "System Default Communications",
+  "audio-device-default": "Standardeinstellung",
+  "audio-device-unknown": "Unbekanntes Gerät",
+
+  "button-test-ok": "OK",
+  "button-text-close": "Schließen",
+  "button-text-dont-tell-me-again": "Sag mir das nicht nochmal",
+  "button-text-lets-do-this": "Lass uns starten!!",
+  "button-text-lets-go": "Los geht's!!",
+  "button-text-not-now": "Nicht jetzt",
+
+  "label-about": "Über",
+  "label-desktop-settings": "Desktop-Einstellungen",
+  "label-donate": "Spenden",
+  "label-help": "Hilfe",
+  "label-issues": "Probleme",
+  "label-learn-more": "Mehr dazu",
+  "label-loading-devices": "Lade Geräte...",
+  "label-pause-after-song": "Nach diesem Titel pausieren",
+  "label-quit": "Beenden",
+  "label-show-lyrics": "Liedtext anzeigen (Beta)",
+  "label-unknown-artist": "Unbekannter Interpret",
+  "label-unknown-song": "Unbekannter Titel",
+  "label-version": "Version",
+
+  "lastfm-login-authorized": "Autorisiert",
+  "lastfm-login-authorizing": "Autorisiere",
+  "lastfm-login-button-text": "Bei Last.FM anmelden",
+  "lastfm-login-error": "Du hast den Autorisierungsvorgang verlassen. Wenn das ein ein Fehler war, versuche es nochmal",
+  "lastfm-login-not-authorized": "Nicht autorisiert",
+  "lastfm-logout-button-text": "Von Last.FM abmelden",
+
+  "lyrics-failed-message": "Entschuldige, aber wir konnten keinen Liedtext finden",
+  "lyrics-loading-message": "Lade Liedtext...",
+  "lyrics-no-song-message": "Aktuell spielt kein Titel",
+
+  "message-pausing-after-song": "Pausieren nach diesem Titel.",
+  "message-pausing-after-song-button": "Nicht nach nach diesem Titel pausieren",
```

Comment:
Yes, indeed. I just wrote that whole thing from top to bottom and read it once afterwards. Thanks!